statsmodels/stats/tests/test_pairwise.py

```diff
@@ -365,4 +365,4 @@ def setup_class(cls):
         cls.reject2 = np.array([False, False, False,  True, False, False,  True, False,  True, False])
 
     def test_hochberg_intervals(self):
-        assert_almost_equal(self.res.halfwidths, self.halfwidth2, 14)
+        assert_almost_equal(self.res.halfwidths, self.halfwidth2, 4)
```

Comment:
The tolerance for this assertion seems like it might have been a continuity test, so it seems appropriate to adjust it since there is an underlying change.